components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3151,14 +3151,18 @@ void RewardsServiceImpl::SaveExternalWallet(const std::string& wallet_type,
   new_wallet.SetStringKey("one_time_string", wallet->one_time_string);
   new_wallet.SetStringKey("user_name", wallet->user_name);
   new_wallet.SetBoolKey("transferred", wallet->transferred);
+  new_wallet.SetStringKey("verify_url", wallet->verify_url);
+  new_wallet.SetStringKey("add_url", wallet->add_url);
+  new_wallet.SetStringKey("withdraw_url", wallet->withdraw_url);
+  new_wallet.SetStringKey("account_url", wallet->account_url);
```

Comment:
I assume these have to be added to iOS as well? Current changes for iOS only include updating the client method

Reply:
correct we need to add it to iOS as well

Reply:
Am in the middle of a 1.11.x build for rewards internals but can add it myself later. If you want to give it a go you will need to update the methods in `BATExternalWallet+DictionaryValue.mm`

Reply:
sure let me do it